duckduckgo.safariextension/html/global.html

```diff
@@ -219,7 +248,15 @@
             }
         }, false);
     }
-
+    
+    safari.application.addEventListener('navigate', function(evt) {
+        if(evt.target && evt.target.url){
+            var urlMatch = evt.target.url.match(/https?:\/\/duckduckgo.com\/\?q=.*/);
```

Comment:
The pattern we used in FF/Chrome was https://duckduckgo.com/?*
https://github.com/duckduckgo/firefox-zeroclickinfo/pull/68/files#diff-37db317c80b52c5e2e654df44f106a3aR25

Maybe this should be the same for consistency?